python/hawkey/query-py.cpp

```diff
@@ -720,8 +720,9 @@ filter_userinstalled(PyObject *self, PyObject *args, PyObject *kwds)
     return final_query;
 }
 
+
 static PyObject *
-filter_unneeded(PyObject *self, PyObject *args, PyObject *kwds)
+filter_UnneededOrSafeToRemove(PyObject *self, PyObject *args, PyObject *kwds, bool SafeToRemove)
```

Comment:
functions names in this file do not use CamelCase. For consistency this should be named filter_unneeded_or_safe_to_remove.